generate_binary.bat

```diff
@@ -38,7 +38,9 @@ copy %ASSETS_DIR_SRC%\mel_filters.npz %ASSETS_DIR_DEST%
 copy %ASSETS_DIR_SRC%\gpt2.tiktoken %ASSETS_DIR_DEST%
 
 REM Code for zipping the final package
-"%WINRAR%" a -r -ep1 -df "%ZIP_FILE_DIR%" "%ZIP_LOCATION%" 
+ECHO Zipping file...
+ "%WINRAR%" a -r -ep1 -df -ibck "%ZIP_FILE_DIR%" "%ZIP_LOCATION%" 
```

Comment:
With this option UI is being used, albeit it is minimized in the task bar. If an incorrect switch is provided it results in an error pop up that needs to be clicked on.

Reply:
Started using rar.exe instead of winrar.exe to avoid UI display.